app/services/billable_metrics/aggregations/sum_service.rb

```diff
@@ -86,7 +86,11 @@ def compute_pay_in_advance_aggregation
       end
 
       def compute_per_event_aggregation
-        events_scope(from_datetime:, to_datetime:).pluck(Arel.sql("COALESCE((#{sanitized_field_name})::numeric, 0)"))
+        if billable_metric.recurring?
+          recurring_events_scope(to_datetime:).pluck(Arel.sql("COALESCE((#{sanitized_field_name})::numeric, 0)"))
```

Comment:
Do we take all events from the beginning here?

I'm not sure to understand this one, does it means that pro-rata will be applies on previous events as well ?